OneCommunityGlobal/HighestGoodNetworkApp
# Nikita create components for input methods for forms

# Description
![image](https://github.com/user-attachments/assets/944a7a56-68a5-4cbd-8127-7dbd7b96f2ac)

## Related PRS (if any):
This frontend PR is not related to any  PR.
Use Developemt branch of backend
…

## Main changes explained:
- Implemented the requirements
- **/form** -- a placeholder webpage to showcase the question maker components
- **/formviewer** -- a placeholder webpage to showcase question answering components

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. go to "http://localhost:3000/form" to test the question maker components
6. go to "http://localhost:3000/formviewer" to test the question maker components
7. verify that all requirements are implemented


## Screenshots or videos of changes:

https://github.com/user-attachments/assets/8d2d8637-473a-4b5e-8a2b-a6d0c6f52f70
